README.md

```diff
@@ -88,13 +89,71 @@ Additionally, for the development we use:
 
 > See the [Development](#development) section for details on how to install `mockery`.
 
-# Installation
+# Quick-Start Installation
 
-## From binaries
+We provide an installation script to automatically install and update the latest version of Trento.
+
+## Trento Server Installation
+
+T.B.D.
+
+## Trento Agent Installation
+
+You can `curl | bash` if you want to live on the edge.
+
+```
+$ curl -sfL https://raw.githubusercontent.com/trento-project/trento/main/install.sh | sudo bash
```

Comment:
This installation worked fine on 15 SP3 , unlike https://github.com/trento-project/trento/pull/226/files#r706322513 on 15 SP2.  

On the successful 15 SP3 installation I then started the trento agent service WITHOUT the server running, this left systemd in a long timeout as the agent tried to reach the server.  Is it possible to add some feedback from the service to systemctl to know the agent is retrying its connection?  Or maybe we can reduce the timeout since it seemed a long time?

Reply:
I modified the script and the README to clarify that the server must be started before the client.

I wouldn't mess with the default consul retry timeouts just for this. If the user wants to know more about what a systemd unit is doing,  the obvious way would be to go and read the journal.